package/yast2-ruby-bindings.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-ruby-bindings
-Version:        3.1.52
+Version:        3.1.51.2
```

Comment:
Already discussed in other PR, basically we use the caasp one or the minor one because if not it will not be available.